Makefile.am

```diff
@@ -92,6 +94,7 @@ libexec_PROGRAMS += eos-updater-avahi/eos-updater-avahi
 
 eos_updater_avahi_eos_updater_avahi_CPPFLAGS = \
 	$(AM_CPPFLAGS) \
+	-DG_LOG_DOMAIN=\""EosUpdaterAvahi"\" \
```

Comment:
Nitpick: Similarly, I’d prefer this to be `eos-updater-avahi`, so it matches the binary name.